### DISCUSSION TITLE
govc datastore.upload - Error: 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by dkirrane** [vmware/govmomi#3113] [Community Help]
`govc datastore.upload` fails with

```
[26-04-23 15:24:25] Uploading... Error: 500 Internal Server Error
govc: 500 Internal Server Error
```

I've tried enable debug but don't see any errors in the `*.res.xml` files in the debug output.
Anything else I can look at to help find the reason for the 500 error?

**Comment by HakanSunay** [comment]
How did you "enable debug" exactly? Do you think you can provide steps to reproduce this issue?
